testsuite/classlibrary/TestVolume.sc

```diff
@@ -0,0 +1,40 @@
+TestVolume : UnitTest {
+
+	var server = Server.default;
```

Comment:
this creates a parse error. Did you test the UnitTest?

Reply:
I started to look into this and got this far (see below). However, I got 
```
PASS:a TestVolume:test_setVolume - initial volume is 0 db
*** ERROR: SynthDef volumeAmpControl2 not found
FAILURE IN SERVER /s_new SynthDef not found
PASS:a TestVolume:test_setVolume - Server volume synth exists
FAILURE IN SERVER /n_set Node 1000 not found
FAILURE IN SERVER /s_get Node 1000 not found
ERROR: Message '-' not understood.
```

now I need to sleep... so leave this to others...

```
TestVolume : UnitTest {

	test_setVolume {
		var server = Server.default;
		var serverVolume = server.volume.volume;
		var ampSynthVolume;

		this.bootServer;

		this.assert(serverVolume == 0, "initial volume is 0 db");

		server.volume.volume = -36;
		server.sync;
		this.assert(server.volume.ampSynth.notNil, "Server volume synth exists");

		server.volume.ampSynth.get(\volumeAmp, { |level| ampSynthVolume = level });
		server.sync;
		this.assertFloatEquals(ampSynthVolume, serverVolume, "volume level correctly set");

		server.volume.reset;
		server.sync;
		server.quit;
	}

// [...]
}
```